config/database.yml

```diff
@@ -89,4 +89,4 @@ test:
 production:
   <<: *default
   database: david_runger_production
-  url: <%= ENV.fetch('DATABASE_URL', nil) %>
+  url: <%= ENV.fetch('DATABASE_URL', Rails.env.production? ? fail('Missing DATABASE_URL!') : nil) %>
```

Comment:
(Mostly unrelated change, but an opportunity for improvement that I saw while looking at this.)

Reply:
Actually, it wasn't a good idea. Fixed here: https://github.com/davidrunger/david_runger/pull/6429 .

I feel like I might have tried to do this before...